cmd/sonobuoy/app/worker.go

```diff
@@ -91,11 +94,17 @@ func runGatherSingleNode(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	client, err := getHTTPClient(cfg)
```

Comment:
shouldn't we retry on the client due to the flakes we are seeing?  There is a separate issue on this but just a PSA.

Reply:
the retry is handled by Pester in pkg/worker/request.go